.gitignore

```diff
@@ -59,3 +59,7 @@ _inc/jetpack-strings.php
 /vendor/composer/
 /vendor/autoload.php
 /vendor/autoload_packages.php
+
+# Tunneling scripts
+/bin/jtuber.sh
+/bin/tunnel.sh
```

Comment:
Could you move those higher on the file, before this line:

`# Don't list files below this comment if they need to be ignored for bundling the Jetpack release.`